test/documentation/seqan3.css

```diff
@@ -199,6 +199,91 @@ div.solution {
 }
 
 #nav-tree .item {
-    padding-bottom: 7px;
-    line-height: 25px;
+    padding-bottom: 3px;
+    line-height: 18px;
+    padding-top: 2px;
+}
+
+#nav-tree .selected {
+    text-shadow: none;
+    background-image: none;
+    color: #1E227D;
+    background-color: rgb(193, 201, 225);
+}
+
+/* Side nav bar styles: */
+#nav-tree .label a { /*default*/
+    color: #3D578C;
+    font-size: 10pt;
+}
+
+/* make all important pages bold */
+#nav-tree .label a[class^="modules"][class$="html"] {
```

Comment:
Please merge them all into one CSS block. They all have the same color and font-weight.